gradle.properties

```diff
@@ -7,9 +7,9 @@
 # https://www.jetbrains.com/intellij-repository/snapshots
 
 pluginName=Grammar-Kit
-pluginVersion=2022.3.2
-pluginSinceIdeaBuild=223
-ideaVersion=2022.3.3
+pluginVersion=2024.0.1
+pluginSinceIdeaBuild=241
```

Comment:
Why make it compatible with 241.* only?

Reply:
I misunderstood the purpose of this property. I reverted the changes, they are not needed for my changes to works